inspectors/models/notation/mmrests/mmrestsettingsmodel.cpp

```diff
@@ -0,0 +1,42 @@
+#include "mmrestsettingsmodel.h"
+
+MMRestSettingsModel::MMRestSettingsModel(QObject* parent, IElementRepositoryService* repository)
```

Comment:
I should mention I am completely unfamiliar with this newfangled model stuff, so I'm learning as I go.  I hope someone more familiar can comment on anything relating to that or to QML specifically.